misc/makezip2.sh

```diff
@@ -58,7 +58,8 @@ cp build/fonts/var/InterVariable-Italic.ttf   "$ZIPDIR/InterVariable-Italic.ttf"
 cp build/fonts/static/Inter*.woff2            "$ZIPDIR/web/" &
 cp build/fonts/var/InterVariable.woff2        "$ZIPDIR/web/InterVariable.woff2"
 cp build/fonts/var/InterVariable-Italic.woff2 "$ZIPDIR/web/InterVariable-Italic.woff2"
-cp misc/dist/inter.css                         "$ZIPDIR/web/"
+cp misc/dist/inter.css                        "$ZIPDIR/web/"
+cp build/fonts/googlefonts/Inter*.ttf         "$ZIPDIR/googlefonts/"
```

Comment:
(Please revert this change)

Was this accidental? This script (makezip2.sh) is used to create the offician distribution, so it shouldn't contain Google Fonts files.

If the intention was to get a zip file of the `googlefonts` make target, I suggest adding a custom `zip ...` invocation to the `googlefonts` make target.

Reply:
(Not affiliated with the Google Fonts team)

Why are there `makezip.sh` and `makezip2.sh`? Is the former obsolete?

Reply:
@mon-jai something like that, I forget. None of them are intended to be used directly, just to reduce the complexity of the makefile.

Reply:
Maybe delete the obsolete one too?

Reply:
Thanks for the review.

This change is intentional. In order for the fonts to be included in Google Fonts, they must reside in a zip file that's attached to the release. The easiest way for me to do this is to bundle the gf edition fonts into the existing zip. If you want them in a separate zip, I can tweak the github action. Either way, whenever you cut a new version, we'll need the googlefonts fonts to be generated and included with the release.

Reply:
Got it! Thanks for clarifying.
Github actions is only used for CI. The actual release I do manually, so, we can add a second zip file e.g. "Inter-VERSION-GoogleFonts.zip" which can then be fetched from github releases (scripted or manually.)
I'll add a patch in a separate comment for you.